com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/TemporalAntiAliasing.shader

```diff
@@ -189,19 +259,22 @@ Shader "Hidden/HDRP/TemporalAA"
             float colorLuma = GetLuma(filteredColor);
             float historyLuma = GetLuma(history);
 
-#if ANTI_FLICKER_MV_DEPENDENT || VELOCITY_REJECTION
-            float motionVectorLength = length(motionVector);
-#else
             float motionVectorLength = 0.0f;
+            float motionVectorLenInPixels = 0.0f;
+
+#if ANTI_FLICKER_MV_DEPENDENT || VELOCITY_REJECTION
+            motionVectorLength = length(motionVector);
+            motionVectorLenInPixels = motionVectorLength * length(_InputSize.xy);
```

Comment:
good idea to precompute length(_InputSize.xy) and pass it as a constant

Reply:
I  think at the time (I just pulled this out of a function here) I measured that the cost of fetching and waiting for the cbuffer was higher than the extra ALU work here. 

I need to do an optimization pass later anyway